File: ios/SDK/Counter.swift
import Foundation

public class Counter {

  var countValue = 5

  open func getCountValue() -> Int {
    return self.countValue
  }

  open func increaseCount() {
    self.countValue += 1
  }

  open func decreaseCount() {
    self.countValue -= 1
  }

  open func setCountValue(_ value: Int) {
    self.countValue = value
  }
}

File: android/app/src/main/java/com/app/sdk/Counter.java
package com.app.sdk;

public class Counter {
    int countValue = 5;

    public int getCountValue() {
        return this.countValue;
    }

    public void increaseCount() {
        this.countValue++;
    }

    public void decreaseCount() {
        this.countValue--;
    }

    public void setCountValue(int countValue) {
        this.countValue = countValue;
    }
}
